### PR TITLE
Quarantines /vg/station due to coronavirus

### PR DESCRIPTION
**Pull request by KingofHONK**
breaks the game!

### CODE REVIEW

**Comment by cloutcode** [comment]
does this finally remove box from rotation

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ env:
   global:
   - BYOND_MAJOR="512"
   - BYOND_MINOR="1453"
-  - ALL_MAPS="tgstation metaclub defficiency packedstation roidstation wheelstation nrvhorizon test_box test_tiny"
+  - ALL_MAPS="tgstation metaclub defficiency packedstation roidstation bagelstation test_box test_tiny"
```
Comment (cloutcode) [inline .travis.yml:11]: the change that saved /vg/ station

**Comment by KingofHONK** [comment]
looks like its removing EVERYTHING from the rotation

**Comment by yclatious** [comment]
nipls what the fuk
Local honkie breaks the fucking server in his quest for bananas. Let's see if the colabs will merge it and mercy kill us all.

**Comment by DrSnips** [comment]
WHOOOPS ALL REVERT BERRIES

**Comment by boy2mantwicethefam** [comment]
how

**Comment by Exxion** [comment]
What the fuck did you do

**Comment by Kurfursten** [comment]
As a joke, I let this stay open for 20 minutes after noticing it

**Comment by Tokanova** [comment]
shoulda merged it. as a joke. haha

**Pull request closed without merging**